src/components/Builder/Contact/Contact.scss

```diff
@@ -0,0 +1,19 @@
+.contact {
+    // width: 20vw;
+    margin-top: 10vh;
+
+    @media screen and (max-width: $breakpoint-mobile) {
+      // display: flex;
+      // margin: auto;
+      text-align: center;
+    }
+
+    &__title {
+      font-weight: bold;
+      font-size: 1.5vw;
+
+      @media screen and (max-width: $breakpoint-mobile) {
+        font-size: 5vw;
+      }
+    }
+}
```

Comment:
Github likes new lines at end of files
<img width="351" alt="Screen Shot 2021-01-26 at 9 19 21 AM" src="https://user-images.githubusercontent.com/13306885/105879942-9e095e00-5fb7-11eb-8564-b0dd4a1b53af.png">